core/src/main/resources/hudson/slaves/Messages.properties

```diff
@@ -23,7 +23,7 @@
 RetentionStrategy.Always.displayName=Keep this agent online as much as possible
 RetentionStrategy.Demand.displayName=Bring this agent online when in demand, and take offline when idle
 RetentionStrategy.Demand.OfflineIdle=Offline because computer was idle; it will be relaunched when needed.
-JNLPLauncher.displayName=Launch agent via Java Web Start
+JNLPLauncher.displayName=Launch agent by connecting it to the master
```

Comment:
```suggestion
JNLPLauncher.displayName=Launch agent by connecting it to the master TCP agent port
```

Reply:
I guess -0. Seems excessively technical. From the user PoV, either I am asking Jenkins to start an agent, typically with SSH; or I am connecting an agent to Jenkins, and the fact that there is a “TCP port” involved in that is usually uninteresting, unless you ran into some firewall issues forcing you to set a fixed port. Anyone else feel strongly one way or the other?

Reply:
FWIW this is trying to tie it to the UI option to enable the TCP agent listener port.

Reply:
Right. But note that the launcher option does not even appear until you enable the port.

Reply:
I don't have much opinion one way or the other. I tend to think that adding "TCP agent port" doesn't help much, but it's also not totally clear what "Launch agent by connecting it to the master" means either. I've tried to come up with better wording but haven't had success yet.